src/caret_analyze/architecture/architecture.py

```diff
@@ -390,6 +431,19 @@ def insert_variable_passing(self, node_name: str,
 
     def remove_publisher_callback(self, node_name: str,
                                   publish_topic_name: str, callback_name: str) -> None:
+        """
+        Remove the association of callback with publisher in "node_name" node.
```

Comment:
```suggestion
        Remove association of callback with publisher in "node_name" node.
```

Reply:
Fix it in [6cf0285](https://github.com/tier4/CARET_analyze/pull/300/commits/6cf0285fa738613d5ed20fe836eed60ce682205c).